src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/v2/NotificationExceptionWithStack.sarif

```diff
@@ -76,10 +78,11 @@
                       "location": {
                         "physicalLocation": {
                           "fileLocation": {
-                            "uri": "file:///C:/src/foobar.cs"
+                            "uri": "file:///C:/src/testtarget.cs"
                           }
                         },
-                        "fullyQualifiedLogicalName": "ExecutionEngine.Engine.FooBar"
+                        "fullyQualifiedLogicalName": "ExecutionEngine.Engine.TestTarget",
+                        "logicalLocationIndex": 0
```

Comment:
>logicalLocationIndex [](start = 25, length = 20)

`ExecutionEngine.Engine.TestTarget` is at index 2, not 0.

Reply:
This indicates a very serious problem somewhere. Issue marked as 'urgent'.

[https://github.com/Microsoft/sarif-sdk/issues/1179](https://github.com/Microsoft/sarif-sdk/issues/1179)


---
In reply to: [243858528](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243858528) [](ancestors = 243858528)